platform/ios/src/MGLMapView.h

```diff
@@ -926,6 +926,23 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (MGLMapCamera *)cameraThatFitsCoordinateBounds:(MGLCoordinateBounds)bounds edgePadding:(UIEdgeInsets)insets;
 
+/**
+ Returns the camera that best fits the given coordinate bounds, optionally with
+ some additional padding on each side.
```

Comment:
This sentence should indicate that the method lets the developer specify a direction and pitch.